api/Document.json

```diff
@@ -2696,10 +2696,12 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": true
+              "version_added": true,
+              "notes": "No longer able to return <code>null</code> as of Firefox 62; see <a href='https://bugzil.la/819475'>Bug 819475</a>."
```

Comment:
I thought this was a bit vague. After reading the bug, I _think_ I get it. What do you think of this? `From Firefox 62, if the domain cannot be identified, then <code>domain</code> is an empty string instead of <code>null</code>. See <a href='https://bugzil.la/819475'>bug 819475</a>.`